cmd/node-cache/app/configmap.go

```diff
@@ -147,6 +149,12 @@ func (c *CacheApp) initDNSConfigSync() {
 	var kubeDNSChan, NodeLocalDNSChan <-chan *config.Config
 	initialKubeDNSConfig := &config.Config{}
 
+	if c.params.KubednsCMPath == "" {
+		if _, err := os.Stat(DefaultKubednsCMPath); !os.IsNotExist(err) {
```

Comment:
If kube-dns configmap is not mounted at the time that node-local-dns starts up, this will not kick off the periodic sync at all. That would be a regression in behavior.
Was the behavior verified on a kube-dns cluster?

Reply:
@pacoxu